bodymass.py

```diff
@@ -0,0 +1,19 @@
+kilo = float (input("kilonuzu giriniz(örnek: 84.9): "))
+boy = float (input("Boyunuzu m cinsinden giriniz: "))
+
+vki = (kilo / (boy**2))
```

Comment:
Please use a standard reading language.
# Please use **ENGLISH** language for writing programs with python, obviously.